src/webui/www/private/scripts/client.js

```diff
@@ -643,12 +643,19 @@ window.addEvent('load', function() {
             $('TotalQueuedSize').set('html', window.qBittorrent.Misc.friendlyUnit(serverState.total_queued_size, false));
         }
 
-        if (serverState.connection_status == "connected")
-            $('connectionStatus').src = 'images/skin/connected.svg';
-        else if (serverState.connection_status == "firewalled")
-            $('connectionStatus').src = 'images/skin/firewalled.svg';
-        else
-            $('connectionStatus').src = 'images/skin/disconnected.svg';
+        switch(serverState.connection_status) {
```

Comment:
please refer to https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#b-other-code-blocks for coding style.